src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.product-card {
+  width: 200px;
```

Comment:
Consider using SCSS variables for main values such as colors, font sizes, and dimensions. This will make your code more maintainable and easier to update.